src/Plugin.php

```diff
@@ -510,14 +550,12 @@ public function checkStates($scan_inactive_and_new_plugins = false, array $exclu
      */
     public function checkPluginState($plugin_key)
     {
-
         $plugin = new self();
 
-        $informations = $this->getInformationsFromDirectory($plugin_key);
+        $information = $this->getPluginInformation()[$plugin_key];
```

Comment:
A plugin can be present in DB but not on filesystem.

```suggestion
        $information = $this->getPluginsInformation()[$plugin_key] ?? [];
```